development-vm/README.md

```diff
@@ -1,276 +1,212 @@
-again.
-```
+This will also run all of the dependencies defined in the Pinfile.
 
-it looks like a problem with this specific version of Vagrant. Using version 1.8.6 works instead: https://releases.hashicorp.com/vagrant/1.8.6/
-You can find more information about this issue here: https://github.com/mitchellh/vagrant/issues/8002
+If you don't need an optional dependency, you can pass the `-w` option:
 
-#### `librarian:install` fails due to permission errors
+    dev$ bowl whitehall -w mapit
 
-Seeing `chown` / `OperationNotPermitted` errors during the `librarian:install` rake task?
+## 9. Keep your VM up to date
 
-Try `vagrant provision` on your host machine, as above.
+There are a few scripts that should be run regularly to keep your VM up to date. In `govuk-puppet/development-vm` there is `update-git.sh` and `update-bundler.sh` to help with this. Also, `govuk_puppet` should be run from anywhere on the VM regularly.
 
-#### Hostname provisioning errors
+The following invocation will do all of this for you.
 
-If you use Vagrant v1.8.x, you may encounter an error along these lines during provisioning:
+    dev$ cd /var/govuk/govuk-puppet/development-vm
+    dev$ ./update-all.sh
 
-```
-Default: Setting hostname...
-/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/plugins/guests/ubuntu/cap/change_host_name.rb:37:in `block in init_package': unexpected return (LocalJumpError)
-    from /opt/vagrant/embedded/gems/gems/vagrant-1.8.1/plugins/communicators/ssh/communicator.rb:222:in `call’
-```
+This will run:
 
-Updating to the latest version (v1.8.5+) might resolve this error. If not, try `sudo vim`ing (or `sudo nano`, whichever you prefer) into that file and removing the offending `return` line.
+1. `git pull` on each of the applications checked out in `/var/govuk`
+1. `govuk_puppet` to bring the latest configuration to the dev VM
+1. `bundle install` for each Ruby application to install any missing gems
+1. `pip install` to update runtime dependencies for any python apps
+
+## 10. Access the web frontend
+
+Most GOV.UK web applications and services are available via the public internet, on the following forms of URL:
+
+* [http://publisher.dev.gov.uk](http://publisher.dev.gov.uk) (local dev, requires the application to be running)
+* [https://www-origin.integration.publishing.service.gov.uk](https://www-origin.integration.publishing.service.gov.uk) (integration, HTTP basic auth)
+* [https://deploy.staging.publishing.service.gov.uk](https://deploy.staging.publishing.service.gov.uk) (staging, restricted to GDS office IP addresses)
+* [https://alert.publishing.service.gov.uk](https://alert.publishing.service.gov.uk) (production, restricted to GDS office IP addresses)
+
+The basic authentication username and password is widely known, so just ask somebody on your team if you don't know it.
+
+## 11. Troubleshooting
+
+See this page in [the dev manual](#TODO) for troubleshooting tips related to the use of Vagrant and the development VM.
```

Comment:
If you're having trouble with Vagrant or the development VM, there are [troubleshooting tips](#TODO) in the dev manual.